packages/world/test/Factories.t.sol

```diff
@@ -95,12 +95,15 @@ contract FactoriesTest is Test, GasReporter {
     // Confirm the msg.sender is owner of the root namespace of the new world
     assertEq(NamespaceOwner.get(ROOT_NAMESPACE_ID), account);
 
-    // Deploy another world
+    // Deploy a second world
 
     // User defined bytes for create2
-    bytes memory _salt1 = abi.encode(1);
-
-    // Address we expect for World
+    // unchecked for the fuzzing test
+    bytes memory _salt1;
+    unchecked {
```

Comment:
Added unchecked here so that the test doesn't revert on overflow/underflow during the fuzzing test.

Reply:
Is the overall intention here just to generate two different salts?

A more clear way to do this would be to have two salt parameters in the fuzz test, and discard them if the fuzzer generates two that are equal. You can do this with a Foundry [assume](https://book.getfoundry.sh/cheatcodes/assume) at the start of the test: 

```solidity
function testWorldFactory(address account, uint256 salt1, uint256 salt2) public {
  vm.assume(salt1 != salt2);
  ...
)